backend/src/schema/types/type/Category.gql

```diff
@@ -1,13 +1,41 @@
+enum _CategoryOrdering {
+  id_asc
+  id_desc
+  name_asc
+  name_desc
+  slug_asc
+  slug_desc
+  icon_asc
+  icon_desc
+  createdAt_asc
+  createdAt_desc
+  updatedAt_asc
+  updatedAt_desc
+  postCount_asc
+  postCount_desc
```

Comment:
any performance issue with this?

Reply:
Yes, there are. But we need it for the admin page for tags and categories. Removing this would break the cypress tests.